dashboard/serializers.py

```diff
@@ -40,23 +39,8 @@ def serialize(cls, program_enrollment):
         )
         return {
             'id': program.id,
-            'enrollments': edx_user_data.raw_enrollments,
-            'certificates': edx_user_data.raw_certificates,
-            'current_grades': edx_user_data.raw_current_grades,
-            'grade_average': cls.calculate_final_grade_average(mmtrack),
-            'is_learner': cls.is_learner(user, program),
+            'enrollments': cls.serialize_enrollments(edx_user_data.enrollments.queryset),
+            'grade_average': mmtrack.calculate_final_grade_average(),
+            'is_learner': is_learner(user, program),
             'email_optin': user.profile.email_optin,
         }
-
-    @classmethod
-    def is_learner(cls, user, program):
```

Comment:
I understand that `roles.api` is a better place for this, but given that it looks like the only place where we are using it is here, why did you have to refactor this?